packages/wdio-cucumber-framework/src/utils.js

```diff
@@ -92,15 +94,21 @@ export function getUniqueIdentifier (target, sourceLocation) {
  * @param {object} message { type: string, payload: object }
  */
 export function formatMessage ({ payload = {} }) {
-    let message = {
-        ...payload,
+    let content = { ...payload }
+
+    /**
+     * need to convert Error to plain object, otherwise it is lost on process.send
+     */
+    if (payload.error && payload.error.message) {
```

Comment:
I have seen situations where an error was thrown but was missing a message, should we take this into account?